packages/fluentui/perf/types.ts

```diff
@@ -1,9 +1,3 @@
-declare global {
-  interface Window {
```

Comment:
used in both gulp task declaration and implementation - removing to not pollute nodejs global scope